src/ensembl/tests/fixtures/entity-viewer/external-references.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
file name: `external-references` -> `external-reference` (or `externalReference`). In singular.

Reply:
Chose external-reference as there are other files in similar name format